docs/how_to/install_puppet_enterprise.md

```diff
@@ -0,0 +1,107 @@
+# How-To Install Puppet Enterprise
+
+Welcome to the choose your own adventure of installing Puppet Enterprise (PE)!
+This doc will guide you through the process beaker's DSL helpers use to install
+PE for acceptance testing.
+
+Note that this is not a complete documentation of the process, but a general
+overview. There will be specific hiccups for particular platforms and special
+cases. These are not included at this point. The idea is that as we come upon
+them, we will now have a place to document those details, so that we can
+over time bring this to 100% completeness.
+
+# First Things First: What Do You Want to Install?
```

Comment:
It's not clear to me why the documentation for `install_puppet_agent_pe_promoted_repo_on` is inside this document; it seems like that is a specific separate use case, tangentially related to a Puppet Enterprise installation, but ultimately not used in actually installing Puppet Enterprise itself. Am I wrong? I might be wrong...